marconi-chain-index/src/Marconi/ChainIndex/Experimental/Indexers/MintTokenEvent.hs

```diff
@@ -125,29 +156,59 @@ data MintAssetRedeemer = MintAssetRedeemer
   }
   deriving (Eq, Ord, Show)
 
-Lens.makeLenses ''MintTokenEvents
+Lens.makeLenses ''MintTokenBlockEvents
 Lens.makeLenses ''MintTokenEvent
 Lens.makeLenses ''MintTokenEventLocation
 Lens.makeLenses ''MintAsset
 Lens.makeLenses ''MintAssetRedeemer
+Lens.makeLenses ''MintTokenEventConfig
+
+mintAssetAssetId :: Lens' MintAsset C.AssetId
+mintAssetAssetId = lens get set
+  where
+    get mintAsset = C.AssetId (mintAsset ^. mintAssetPolicyId) (mintAsset ^. mintAssetAssetName)
+    set mintAsset (C.AssetId policyId assetName) =
+      mintAsset
+        & mintAssetPolicyId .~ policyId
+        & mintAssetAssetName .~ assetName
+    set mintAsset C.AdaAssetId =
+      mintAsset
+        & mintAssetPolicyId .~ ""
+        & mintAssetAssetName .~ ""
 
 instance SQL.ToRow (Core.Timed C.ChainPoint MintTokenEvent) where
-  toRow te =
-    let snoField = te ^. Core.point . Lens.to (SQL.toField . C.chainPointToSlotNo)
-     in snoField : SQL.toRow (te ^. Core.event)
+  toRow te = SQL.toRow (te ^. Core.point) ++ SQL.toRow (te ^. Core.event)
+
+instance SQL.FromRow (Core.Timed C.ChainPoint MintTokenEvent) where
+  fromRow = do
+    point <- SQL.fromRow
+    event <- SQL.fromRow
+    pure $ Core.Timed point event
```

Comment:
Can we decide whether we want to use do-notation or `<$>` and `<*>` in the straightforward cases? (so that the code for `MintTokenEvent`, `MintTokenEventLocation` and this instances look alike)

Reply:
Just missed it. <$> and <*> is better